### ISSUE TITLE
week 3/ai, crude oil prices

### ISSUE THREAD

**Issue by oem** [romeokienzler/developerWorks#3]
Heya, first of all, thanks for your talk (I am the guy with the dopamin kick, if you remember :) ) ! I got a small problem with plotting the predicted oil prices. For me it looks like the predicted values are a very flat graph:
<img width="1038" alt="screen shot 2018-01-25 at 13 37 23" src="https://user-images.githubusercontent.com/230631/35388737-432000a4-01d5-11e8-8a05-a9752da00578.png">

I figured out what the problem is: the test_set is an array with one scalar vector (dtype=object).
y_test meanwhile is an array with one float64 vector.
One solution would be following:
plt.plot(np.float64(test_set)[timesteps:len(y_test)], color = 'red', label = 'Real Crude Oil Prices')

this fixes it for me:
<img width="1056" alt="screen shot 2018-01-25 at 13 44 17" src="https://user-images.githubusercontent.com/230631/35388880-e198c20c-01d5-11e8-8400-bc59ebc978b1.png">

The other graph further down would need the same treatment.

Doing it in this cell is probably not the ideal location tho.

For reference, I am running:

Python: 3.6.3
Keras: 2.1.3
tensorflow: 1.4.0
